stanford_profile_helper.module

```diff
@@ -219,9 +219,23 @@ function stanford_profile_helper_form_alter(&$form, FormStateInterface $form_sta
     // from the active filters.
     // @link https://www.drupal.org/project/views_bulk_operations/issues/3055770#comment-13116724
     unset($form['header']['views_bulk_operations_bulk_form']['select_all']);
+
+    // Sort the action menu options alphabetically.
+    $actions_array = $form['header']['views_bulk_operations_bulk_form']['action']['#options'];
+    uasort($actions_array, '_stanford_profile_helper_action_sort');
+    $form['header']['views_bulk_operations_bulk_form']['action']['#options'] = $actions_array;
   }
 }
 
+/**
+ * A sorting function for the action array.
+ */
+function _stanford_profile_helper_action_sort($a, $b) {
+  $a = is_a($a, "Drupal\Core\StringTranslation\TranslatableMarkup") ? $a->render() : (string) $a;
```

Comment:
Should use namespaces with the `use` at the top.

Reply:
Fixed.  Thanks.